x-pack/plugins/security_solution/public/entity_analytics/components/asset_criticality/use_asset_criticality.ts

```diff
@@ -59,7 +64,18 @@ export const useAssetCriticalityData = ({
   });
 
   const mutation = useMutation({
-    mutationFn: createAssetCriticality,
+    mutationFn: (params: Params) => {
+      if (params.criticalityLevel === 'unassigned') {
+        return deleteAssetCriticality({ idField: params.idField, idValue: params.idValue });
```

Comment:
Could you add a unit test? :bowtie:

Reply:
Everything else looks good to me!
I tested it and 😍

Reply:
added